tracim/tracim/model/__init__.py

```diff
@@ -15,7 +46,16 @@
 # Base class for all of our model classes: By default, the data model is
 # defined with SQLAlchemy's declarative extension, but if you need more
 # control, you can switch to the traditional method.
-DeclarativeBase = declarative_base()
+convention = {
+  "ix": 'ix__%(column_0_label)s',
+  "uq": "uk__%(table_name)s__%(column_0_name)s",
+  "ck": "ck__%(table_name)s__%(constraint_name)s",
+  "fk": "fk__%(table_name)s__%(column_0_name)s__%(referred_table_name)s",
+  "pk": "pk__%(table_name)s"
+}
+
+metadata = MetaData(naming_convention=convention)
```

Comment:
by the way, why "uq" => "uk" ?
Could you make this more homogeneous ? ("uk" => "uk" or "uq" => "uq")